release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -87,6 +87,25 @@ infrastructure.
 //../installing/installing_openstack/installing-openstack-user.adoc#installing-openstack-user[Installing a cluster on OpenStack on your own infrastructure]
 //../installing/installing_openstack/installing-openstack-user-kuryr.adoc#installing-openstack-user-kuryr[Installing a cluster on OpenStack with Kuryr on your own infrastructure]
 
+[id="ocp-4-4-installing-cluster-on-openstack-no-longer-requires-swift-storage-service"]
+==== Installing a cluster on OpenStack no longer requires the Swift object storage service
```

Comment:
Feel free to take a bat to this item, if you like. I wrote part of it, so I'm not objective.